packages/apps/plugins/plugin-markdown/src/components/EditorMain.tsx

```diff
@@ -82,6 +82,7 @@ const EditorMain = ({ comments, toolbar, extensions: _extensions, ...props }: Ed
           root: {
             className: mx(
               focusRing,
+              'm-0.5 mx-1', // TODO(burdon): Standardize for focusRing. Just focus "paper" of document?
```

Comment:
Right, I think the solution for this problem belongs elsewhere. Glad to take it on if you like, would like to incorporate the feedback here as well: https://discord.com/channels/837138313172353095/837688768872972358/1204725830148685865

Reply:
Come to think of it, if we follow with our principle that the focus indicator is optional when the element will have a blinking cursor inside it, then we should actually look into preventing direct focus of the text area and instead always pass focus within.

Reply:
I wasn’t able to find a comfortable solution for skipping direct focus of the text area, but I think this may be a reasonable compromise for now since EditorMain no longer really has a top- or bottom-edge (that it is also responsible for rendering):

https://github.com/dxos/dxos/assets/855039/1d885fe3-af70-4d24-af6e-d154b995e97d